src/signals/incident/components/form/MapSelectors/Streetlight/StreetlightLayer/StreetlightLayer.tsx

```diff
@@ -0,0 +1,19 @@
+import { useMatchMedia } from '@amsterdam/asc-ui/lib/utils/hooks'
+import { useContext } from 'react'
+import AssetLayer from '../../Asset/Selector/WfsLayer/AssetLayer'
+import ReportedLayer from '../../Asset/Selector/WfsLayer/ReportedLayer'
+import AssetSelectContext from '../../Asset/context'
+
+export const StreetlightLayer = () => {
+  const { meta } = useContext(AssetSelectContext)
+  const [desktopView] = useMatchMedia({ minBreakpoint: 'tabletM' })
+
+  return (
+    <>
+      <AssetLayer featureTypes={meta.featureTypes} desktopView={desktopView} />
+      <ReportedLayer featureTypes={meta.featureTypes} />
```

Comment:
Maybe filter here on feature types that the reported layer should actually render?

Reply:
oke